### PR TITLE
Changes hour select to display only even hours

### LINKED ISSUE #128: Only allow 2 hours periods in planning availability selector
Dans le planing, il faudrait remplacer la sélection d'heure de la période de dispo souhaitée par des  créneaux de deux heures pour début et fin

### PR DESCRIPTION
**Pull request by Aurelie-feyzin**
Closes #128

Un peu bourrin comme solution mais je n'ai pas réussit a utiliser l'option `hours`de DateTimeType de Sf et datepickerrange ne propose pas d'option pour cela.

### CODE REVIEW

File: assets/js/planning.js [hunk 1]
```diff
@@ -43,13 +43,28 @@ function initDatesRange($picker, $from, $to, withTime) {
...
+
   $picker.on('cancel.daterangepicker', function () {
     $picker.val('');
     $from.val('');
     $to.val('');
   });
 }
 
+function hourselectOnlyEvenHour() {
+  let allSelectHours = document.getElementsByClassName('hourselect');
```
Comment (mRoca) [inline assets/js/planning.js:58]: you should use a jQuery selector and look for this item only in the current datepicker

File: assets/js/planning.js [hunk 2]
```diff
@@ -43,13 +43,28 @@ function initDatesRange($picker, $from, $to, withTime) {
...
+  console.log(allSelectHours);
```
Comment (mRoca) [inline assets/js/planning.js:59]: to remove

File: assets/js/planning.js (hunk 1 above)
Comment (mRoca) [reply assets/js/planning.js:58]: e.g.:

```js
function hourselectOnlyEvenHour($picker) {
    const allSelectHours = $picker.find('.hourselect');
}
```

**Comment by mRoca** [comment]
Your select update is not persisted: if you select an hour in the first time select, both are reset with previous even & odd values. It's mainly because they must be re-render in order to disable invalid time in the second one if you select something in the first one

**Comment by mRoca** [comment]
And you have forgotten to handle the previous selected values : try to define a search, then to open the selector. The previously chosen value should be selected

**Comment by mRoca** [comment]
You can find the widget source code there if you need it : https://github.com/dangrossman/daterangepicker/blob/master/daterangepicker.js

**Comment by Aurelie-feyzin** [comment]
PR ouverte directement sur daterangepicker : https://github.com/dangrossman/daterangepicker/pull/2079

**Pull request closed without merging**